packages/picasso.js/src/core/chart-components/axis/axis.js

```diff
@@ -22,10 +22,13 @@ function alignTransform({ align, inner }) {
 function resolveAlign(align, dock) {
   const horizontal = ['top', 'bottom'];
   const vertical = ['left', 'right'];
-  if (horizontal.indexOf(align) !== -1 && vertical.indexOf(dock) === -1) {
-    return align;
-  }
-  if (vertical.indexOf(align) !== -1 && horizontal.indexOf(dock) === -1) {
+
+  const docksHorizontally = horizontal.includes(dock);
+  const docksVertically = vertical.includes(dock);
+  const alignsHorizontally = horizontal.includes(align);
+  const alignsVertically = vertical.includes(align);
+
+  if ((alignsHorizontally && !docksVertically) || (alignsVertically && !docksHorizontally)) {
```

Comment:
I thinking this could be simplified to something like this:
```js
const validAlign = vertical.some(v === dock) && vertical.some(v === align);

return validAlign ? align : dock;
```

Reply:
I think I will leave that refactoring part for this PR.